src/main/scala/mesosphere/marathon/MarathonSchedulerDriver.scala

```diff
@@ -3,7 +3,6 @@ package mesosphere.marathon
 import java.io.FileInputStream
 
 import com.google.protobuf.ByteString
-import mesosphere.chaos.http.HttpConf
```

Comment:
Out of curiosity - this was an unused import? Or why is it the only change in this file?

Reply:
HttpConf was moved to package `mesosphere.marathon`, so import was no longer needed.